src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String line_separator = "     _______________________\n";
+    static final String spacing = "     ";
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        int MAXIMUM = 100;
+        Task[] task_list = new Task[MAXIMUM];
+        int task_num = 0;
```

Comment:
Variable names could follow the camelCase standard, eg taskList and taskNum instead of task_list and task_num. Otherwise I think that the namings are quite clear so good job